spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/ajax/ImportExportControllerHelperService.java

```diff
@@ -169,7 +171,7 @@ public JSONObject confirmSamplesUpload(HttpSession session, JSONObject json) thr
     for (JSONArray jsonArrayElement : (Iterable<JSONArray>) jsonArray) {
 
       Sample s = null;
-      if (jsonArrayElement.get(3) != null && !"".equals(jsonArrayElement.getString(3))) {
+      if (jsonArrayElement.get(3) != null && !isStringEmptyOrNull(jsonArrayElement.getString(3))) {
```

Comment:
yet again, very out of scope.
Magic ints, should be defined as constants... somewhere.